src/current/cockroachcloud/export-metrics.md

```diff
@@ -214,11 +216,118 @@ A subset of CockroachDB metrics require that you explicitly [enable percentiles]
 
 </section>
 
+<section class="filter-content" markdown="1" data-scope="prometheus-metrics-export">
+
+1. Find your CockroachDB {{ site.data.products.dedicated }} cluster ID:
+
+	1. Visit the CockroachDB {{ site.data.products.cloud }} console [cluster page](https://cockroachlabs.cloud/clusters).
+	1. Click on the name of your cluster.
+	1. Find your cluster ID in the URL of the single cluster overview page: `https://cockroachlabs.cloud/cluster/{your_cluster_id}/overview`. It should resemble `f78b7feb-b6cf-4396-9d7f-494982d7d81e`.
+
+1. Issue the following Cloud API command to enable metrics export for your CockroachDB {{ site.data.products.dedicated }} cluster:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~shell
+    curl --request POST \
+      --url https://cockroachlabs.cloud/api/v1/clusters/{cluster_id}/metricexport/prometheus \
+      --header "Authorization: Bearer {secret_key}"
+    ~~~
+
+    Where:
+    - `{cluster_id}` is your CockroachDB {{ site.data.products.dedicated }} cluster ID as determined in step 1, resembling `f78b7feb-b6cf-4396-9d7f-494982d7d81e`.
+    - `{secret_key}` is your CockroachDB {{ site.data.products.dedicated }} API key. See [API Access]({% link cockroachcloud/managing-access.md %}) for instructions on generating this key.
+
+1. Depending on the size of your cluster and how many regions it spans, the configuration may take a moment. You can monitor the ongoing status of the configuration using the following Cloud API command:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~shell
+    curl --request GET \
+      --url https://cockroachlabs.cloud/api/v1/clusters/{cluster_id}/metricexport/prometheus \
+      --header "Authorization: Bearer {secret_key}"
+    ~~~
+
+    Run the command periodically until the command returns a `status` of `ENABLED`, at which point the configuration across all nodes is complete. The response also includes `targets`, a map of scrape endpoints exposing metrics to regions. For example:
+
+    ~~~
+    {
+      "cluster_id": "f78b7feb-b6cf-4396-9d7f-494982d7d81e",
+      "user_message": "This integration is active.",
+      "status": "ENABLED",
+      "targets": {
+        "us-east4": "https://cockroachlabs.cloud/api/v1/clusters/f78b7feb-b6cf-4396-9d7f-494982d7d81e/metricexport/prometheus/us-east4/scrape"
+      }
+    }
+    ~~~
+
+    There is a separate scrape endpoint per region if you have a multi-region cluster.
```

Comment:
"multi-region" could be a link to https://www.cockroachlabs.com/docs/stable/multiregion-overview

Reply:
added link